src/org/lsst/ts/jenkins/components/Csc.groovy

```
@@ -93,10 +105,12 @@ def build_salobj_conda(label, concatVersion) {
     """
 }
 
-def upload_conda(name, label) {
+def upload_conda(name, label, arm, noarch) {
     // Upload the conda package
     // Takes the name of the package and a label
-    if((name.contains("ts-salobj")) || (name.contains("ts-idl"))) {
+    if(arm) {
+        arch = "arm-64"
+    } else if(noarch) {
```

It would be better to introduce one parameter called "arch" which may contain the values "arm-64", "noarch" and "linux-64" and then simply let the build fail if the parameter contains a different value. Also, I don't really see why you chose the value "arm-64" since the conda architecture is called "aarch64". Please use that instead to avoid confusion with the new arm64 architecture for MacOS.